R/Report_StudyInfo.R

```diff
@@ -40,7 +40,6 @@ Report_StudyInfo <- function(
     Value = unname(unlist(lStudy))
   ) %>%
     dplyr::left_join(dfLabels, by = "Param") %>%
-    dplyr::select("Description", "Value") %>%
```

Comment:
With only 2 columns left, is there a way to choose the studyid, nickname, and site/participants numbers just from `Description`? Looking at the testthat file, it looked like `Param` seemed to have some standard conventions/controlled terminology that lends itself to use a `filter` statement but if the descriptions are always consistent, moving this `select` statement isn't necessary and I can modify the `%in%` criteria of the `filter`

Reply:
@samussiah

Reply:
What you have looks good!  Better to use `Param` to choose the appropriate rows.  It's not controlled terminology exactly but we can assume it'll have these attributes:

- `GroupID`
- `SiteCount`
- `ParticipantCount`

because each of these columns are explicitly defined in the [`data_reporting` workflow](https://github.com/Gilead-BioStats/gsm/blob/dev/inst/workflow/data_reporting.yaml).